tests/PageObjects/EditSongsPage.ts

```diff
@@ -12,6 +12,10 @@ export class EditSongsPagePO {
     await this.page.locator(`[data-test="hide-song"][data-song="${songID}"]`).click();
   }
 
+  public async expectHideSongNotToBeVisible(songID: string) {
```

Comment:
looking at the context where this method it used, more proper name would be `expectSongToBeHidden`